{"uuid": "19b7a01d-6c40-4ea5-8d9b-b1dd2498ed1c", "children": ["0b945832-0d02-4114-991a-8a7681b35274"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "AttributeError: 'FixtureFunctionDefinition' object has no attribute 'reset'\n", "trace": "  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 923, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\pc\\Desktop\\Challenge Gaston Ferreyra\\Mobile-test\\conftest.py\", line 21, in driver\n    driver.reset()\n    ^^^^^^^^^^^^\n"}, "start": 1759529991708, "stop": 1759529991708}], "start": 1759529991708, "stop": 1759529991822}